### ISSUE TITLE
[Feature Request] - Support for .env files with moleculer-runner

### ISSUE THREAD

**Issue by go4cas** [ice-services/moleculer#158]
Including environment variables for components outside of the moleculer modules, e.g. database credentials, external API tokens, etc, etc.

Using a package like [dotenv](https://github.com/motdotla/dotenv) to load the specified `.env` file when moleculer-runner starts up.

This would extend the [env variables loaded by moleculer-runner](http://moleculer.services/docs/runner.html#Environment-variables), when loading the `moleculer.config.js`.

**Comment by icebob** [comment]
I will create `--env` & `--envfile` cli options. 
```sh
# Load the default .env file before broker starting
$ moleculer-runner --env 

# Load the specified .my-env file before broker starting
$ moleculer-runner --envfile .my-env
```

> I won't add `dotenv` to the project as a dependency because Moleculer doesn't use it. You need to install it explicitly in your project with `npm i dotenv`.

Any comment or suggestion?

**Comment by go4cas** [comment]
Couple of questions:

- When you say "Load the .env file", does that mean that the variables declared in this file will be available with `process.env.`? Because, if that is the case, then there is no need for the `dotenv` package
- What will happen if the same variable is declared in the `.env` file and in the `moleculer.config.js` file? Which will take priority?

**Comment by icebob** [comment]
* Yes, it will load it with `dotenv` and can be accessed via `process.env`. Why doesn't need `dotenv` to load .env file?
* I don't implement it. What is your suggestion? What has bigger priority?

**Comment by go4cas** [comment]
I think the `.env` should take priority. But, I assume it will merge the variables from both files?

**Comment by icebob** [comment]
Yes, I will merge them.

**Comment by icebob** [comment]
@go4cas I implemented it in #162

**Comment by icebob** [comment]
Released in 0.11.9
